test/core/crypto/crypto_store/crypto_store_test.cpp

```diff
@@ -113,13 +113,30 @@ struct CryptoStoreTest : public test::BaseFS_Test {
   SR25519Keypair sr_pair;
 };
 
+/**
+ * @given cryptostore instance @and non-existant, but valid directory path
+ * @when call initialize using provided directory as argument
+ * @then initialization succeeds @and corresponding directory is created
+ */
+TEST_F(CryptoStoreTest, InitializeSuccess) {
+  auto path = crypto_store_test_directory / "aaa";
+  ASSERT_FALSE(boost::filesystem::exists(path));
+  EXPECT_OUTCOME_TRUE_MSG_1(crypto_store->initialize(path),
```

Comment:
**EXPECT_OUTCOME_ERROR** would be usefull here =)

Reply:
Isn't used where we think an error should occur?